student.rb

```diff
@@ -0,0 +1,10 @@
+# Student is a class representing a student, which inherits from Person.
+class Student < Person
+  def initialize(age, _classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+  end
+
+  def play_hooky
+    '¯\\(ツ)/¯'
+  end
```

Comment:
As directed, make the classroom setter method and ensure that the student is added to the classroom. Look at [this for more info](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)

Consider the code snippet below:

```ruby
@classroom = classroom
classroom.students.push(self) unless classroom.students.include?(self)
```